libraries/script-engine/src/WebSocketClass.cpp

```diff
@@ -97,6 +103,25 @@ void WebSocketClass::handleOnMessage(const QString& message) {
     }
 }
 
+void WebSocketClass::handleOnBinaryMessage(const QByteArray& message) {
+    if (_onMessageEvent.isFunction()) {
+        QScriptValueList args;
+        QScriptValue arg = _engine->newObject();
+        QScriptValue data = _engine->newVariant(QVariant::fromValue(message));
+        QScriptValue ctor = _engine->globalObject().property("ArrayBuffer");
+        ArrayBufferClass* array = qscriptvalue_cast<ArrayBufferClass*>(ctor.data());
+        QScriptValue arrayBuffer;
+        if (!array) {
+            qWarning() << "WebSocketClass::handleOnBinaryMessage !ArrayBuffer";
```

Comment:
Can you make this warning more helpful and set a proper category with qCWarning ?